Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
Wouldn't it be cleaner to skip build-time symlinking entirely and just commit a relative symlink in the repo?  Git will happily track it. :+1:

I'm curious how the godeps tool handles extra content like this inside the `_workspace` directory -- does it consider that directory to be its personal space and just blast away everything inside?  Does it maybe have an option for creating/keeping this symlink automatically for us?